metadata.json

```diff
@@ -34,8 +34,8 @@
   "requirements": [
     {"name":"puppet","version_requirement":">= 4.1.0"},
     {"name": "ruby","version_requirement": ">=2.2.3"},
-    {"name": "oneview-sdk-ruby","version_requirement": ">=3.0.0"}
+    {"name": "oneview-sdk-ruby","version_requirement": ">=3.1.0"}
   ],
-  "tags": [ "oneview", "hpe", "hewlett packard enterprise", "i3s", "image streamer", "bare-metal provisioning" ],
+  "tags": [ "oneview", "hpe", "hewlett packard enterprise", "icsp", "i3s", "image streamer", "bare-metal provisioning" ],
```

Comment:
...maybe it makes sense, since you can create a server profile with a Deployment Plans created on i3s. I suggest remove the icsp for now